Warning: no access to tty (Bad file descriptor).
Thus no job control in this shell.
Infiniband hardware address can be incorrect! Please read BUGS section in ifconfig(8).
eno1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 128.239.56.64  netmask 255.255.252.0  broadcast 128.239.59.255
        inet6 fe80::c654:44ff:fe55:4d7f  prefixlen 64  scopeid 0x20<link>
        ether c4:54:44:55:4d:7f  txqueuelen 1000  (Ethernet)
        RX packets 342260753  bytes 211324365380 (196.8 GiB)
        RX errors 0  dropped 472  overruns 0  frame 0
        TX packets 888796303  bytes 1242467425802 (1.1 TiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eno2: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        ether c4:54:44:55:4d:80  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

ib0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 65520
        inet 192.168.56.64  netmask 255.255.252.0  broadcast 192.168.59.255
        inet6 fe80::f652:1403:b:f3c0  prefixlen 64  scopeid 0x20<link>
        infiniband 80:00:00:29:FE:80:00:00:00:00:00:00:00:00:00:00:00:00:00:00  txqueuelen 256  (InfiniBand)
        RX packets 772753920  bytes 3361581556675 (3.0 TiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 376855521  bytes 368123767035 (342.8 GiB)
        TX errors 0  dropped 22450 overruns 0  carrier 0  collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 13692152  bytes 162440132325 (151.2 GiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 13692152  bytes 162440132325 (151.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

WARNING:torch.distributed.run:
*****************************************
Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
*****************************************
======================================================
Environment variables set by the agent on PID 31650:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '1',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '1',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31649:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '0',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '0',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31654:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '5',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '5',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31652:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '3',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '3',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31651:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '2',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '2',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31656:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '7',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '7',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31661:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '10',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '10',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31655:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '6',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '6',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31653:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '4',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '4',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31660:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '9',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '9',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31662:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '11',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '11',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

======================================================
Environment variables set by the agent on PID 31657:
{'GROUP_RANK': '0',
 'LOCAL_RANK': '8',
 'MASTER_ADDR': 'vx01.sciclone.wm.edu',
 'MASTER_PORT': '51720',
 'RANK': '8',
 'TORCHELASTIC_MAX_RESTARTS': '0',
 'TORCHELASTIC_RESTART_COUNT': '0',
 'WORLD_SIZE': '12'}
======================================================

On PID 31653, after init process group, rank=4, world_size = 12
On PID 31649, after init process group, rank=0, world_size = 12
On PID 31657, after init process group, rank=8, world_size = 12

On PID 31652, after init process group, rank=3, world_size = 12
On PID 31661, after init process group, rank=10, world_size = 12


On PID 31662, after init process group, rank=11, world_size = 12

On PID 31656, after init process group, rank=7, world_size = 12
On PID 31651, after init process group, rank=2, world_size = 12




On PID 31655, after init process group, rank=6, world_size = 12

On PID 31654, after init process group, rank=5, world_size = 12

On PID 31660, after init process group, rank=9, world_size = 12
On PID 31650, after init process group, rank=1, world_size = 12


trainer_4
trainer_9
trainer_1
trainer_6
trainer_8
trainer_2
trainer_7
trainer_10
trainer_11
trainer_5
trainer_3
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
DONE UP TO HERE IN TRAINING LOOP!!!!
[W NNPACK.cpp:79] Could not initialize NNPACK! Reason: Unsupported hardware.
[W python_anomaly_mode.cpp:104] Warning: Error detected in NativeBatchNormBackward0. Traceback of forward call that caused the error:
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/distributed/rpc/internal.py", line 204, in _run_function
    result = python_udf.func(*python_udf.args, **python_udf.kwargs)
  File "/sciclone/home20/hmbaier/lc_v2/utils2.py", line 71, in _call_method
    return method(rref.local_value(), *args, **kwargs)
  File "/sciclone/home20/hmbaier/lc_v2/utils2.py", line 459, in forward
    out = self.model(lc_im, census, im, muni_id)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/sciclone/home20/hmbaier/lc_v2/models.py", line 175, in forward
    return self.conv_net(glimpse)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/sciclone/home20/hmbaier/lc_v2/models.py", line 63, in forward
    x = self.layer2(x)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/nn/modules/container.py", line 141, in forward
    input = module(input)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torchvision/models/resnet.py", line 71, in forward
    out = self.bn1(out)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/nn/modules/batchnorm.py", line 168, in forward
    return F.batch_norm(
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/nn/functional.py", line 2282, in batch_norm
    return torch.batch_norm(
 (function _print_stack)
WARNING:torch.distributed.elastic.rendezvous.dynamic_rendezvous:The node 'vx01.sciclone.wm.edu_31643_0' has failed to send a keep-alive heartbeat to the rendezvous '790876' due to an error of type RendezvousTimeoutError.
[W tensorpipe_agent.cpp:886] RPC agent for trainer_7 encountered error when reading incoming response from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_7 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:886] RPC agent for trainer_9 encountered error when reading incoming response from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:886] RPC agent for trainer_1 encountered error when reading incoming response from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:886] RPC agent for trainer_4 encountered error when reading incoming response from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:886] RPC agent for trainer_2 encountered error when reading incoming response from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_4 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_8 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_9 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:886] RPC agent for trainer_3 encountered error when reading incoming response from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_11 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_10 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_1 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:886] RPC agent for trainer_5 encountered error when reading incoming response from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_2 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_3 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:886] RPC agent for trainer_6 encountered error when reading incoming response from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_5 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:681] RPC agent for trainer_6 encountered error when reading incoming request from parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #10 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #11 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #12 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #13 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #14 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #15 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #16 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #17 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #18 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #19 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #20 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #21 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #22 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #23 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #24 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #25 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #26 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #27 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #28 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #29 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #30 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #31 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #32 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #33 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #34 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #35 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #36 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #37 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #38 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #39 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #40 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #41 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #42 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #43 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #45 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #44 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #46 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #47 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #48 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #49 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #51 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #52 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #53 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #54 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #55 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #56 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #57 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #58 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #59 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #60 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #61 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #62 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #63 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #64 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #65 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #66 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #67 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #68 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #69 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #70 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #71 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #72 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #73 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #74 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #75 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #76 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #77 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #78 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #79 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #80 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #81 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #82 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #83 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #84 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #85 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31650 closing signal SIGTERM
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #86 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #87 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #88 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #89 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #90 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #91 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #92 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #93 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
[W tensorpipe_agent.cpp:862] RPC agent for trainer_7 encountered error when sending outgoing request #94 to parameter_server: EOF: end of file (this error originated at tensorpipe/transport/uv/connection_impl.cc:132)
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31651 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31652 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31653 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31654 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31655 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31656 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31657 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31660 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31661 closing signal SIGTERM
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 31662 closing signal SIGTERM
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: -9) local_rank: 0 (pid: 31649) of binary: /sciclone/home20/hmbaier/.conda/envs/dhsrl4/bin/python
Traceback (most recent call last):
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/bin/torchrun", line 33, in <module>
    sys.exit(load_entry_point('torch==1.10.0', 'console_scripts', 'torchrun')())
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 345, in wrapper
    return f(*args, **kwargs)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/distributed/run.py", line 719, in main
    run(args)
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/distributed/run.py", line 710, in run
    elastic_launch(
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/distributed/launcher/api.py", line 131, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/sciclone/home20/hmbaier/.conda/envs/dhsrl4/lib/python3.9/site-packages/torch/distributed/launcher/api.py", line 259, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
======================================================
/sciclone/home20/hmbaier/lc_v2/run2.py FAILED
------------------------------------------------------
Failures:
  <NO_OTHER_FAILURES>
------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2022-01-23_16:12:22
  host      : vx01.sciclone.wm.edu
  rank      : 0 (local_rank: 0)
  exitcode  : -9 (pid: 31649)
  error_file: <N/A>
  traceback : Signal 9 (SIGKILL) received by PID 31649
======================================================
